calc/vxlan_resolver.go

```diff
@@ -423,25 +490,80 @@ func (c *VXLANResolver) routeReady(r vxlanRoute) bool {
 		logCxt.Debug("Don't yet know the VTEP for this route")
 		return false
 	}
+
+	if _, ok := c.nodeNameToNode[c.hostname]; !ok {
+		logCxt.Debug("No cidr block yet for local node")
+		return false
+	}
+
+	if _, ok := c.nodeNameToNode[r.node]; !ok {
+		logCxt.Debug("No cidr block yet for node")
+		return false
+	}
+
 	return true
 }
 
+func (c *VXLANResolver) nodeCidr(nodeName string) (*cnet.IPNet, error) {
+	logCxt := logrus.WithField("node", nodeName)
+
+	node := c.nodeNameToNode[nodeName]
+	bgp := node.Spec.BGP
+	if bgp == nil {
+		return nil, errors.New("no bgp information for node")
```

Comment:
I think to make this log more useful for people skimming the logs, we should say "No IPv4 address for node".

It's kinda confusing that we use a field in the BGP section in order to do VXLAN routes, since BGP isn't involved at all. But, that's where we're at. But the real thing we're looking for is the IPv4 address.

Reply:
right, good point. I'll update